static/app/views/dashboards/widgets/timeSeriesWidget/timeSeriesWidgetVisualization.tsx

```diff
@@ -167,14 +166,10 @@ export function TimeSeriesWidgetVisualization(props: TimeSeriesWidgetVisualizati
       registerWithWidgetSyncContext(echartsInstance);
 
       if (hasReleaseBubblesSeries) {
-        createReleaseBubbleHighlighter(echartsInstance);
+        connectReleaseBubbleChartRef(e);
```

Comment:
In this component we already had a callback ref, but for `EventGraph`, we can pass `connectReleaseBubbleChartRef` directly to the component without a callback ref e.g.

```javascript
// imo more clear what it's used for as it calls out "chart" + "ref"
<EventGraph {...props} ref={connectReleaseBubbleChartRef} />
// vs below, less clear why it's passed as ref,  but more explicit about it does
<EventGraph {...props} ref={createReleaseBubbleHighlighter} />
```